conformance/client/google-java/build.gradle.kts

```diff
@@ -16,12 +16,12 @@ plugins {
 tasks {
     compileKotlin {
         kotlinOptions {
-            // Generated Kotlin code for protobufs uses OptIn annotation
+            // Generated Kotlin code for protobuf uses OptIn annotation
             freeCompilerArgs += "-opt-in=kotlin.RequiresOptIn"
         }
     }
     shadowJar {
-        archiveBaseName.set("shadow")
+        archiveFileName.set("conformance-client-java.jar")
```

Comment:
I made this change to make it easier to actually use the output JAR from another target in the `Makefile`. Without this, the actual file has a `<version>-SNAPSHOT` in the filename. But I couldn't find any simple way to figure out the `<version>`, so that a command could reference the correct filename. (It appears to be computed dynamically by Gradle based on git tags.) So this removes the suffix so we can easily reference the correct output JAR file.

Reply:
Another option here is that you can make it use the application plugin (https://docs.gradle.org/current/userguide/application_plugin.html) which when combined with `installDist` will create an executable script with the right jar file name embedded. I do that in protoc-gen-connect-kotlin to make it callable.

Reply:
🤦, doh! I should have done that to begin with -- good idea! I wasn't aware of the application plugin and was mainly used to building "fat jars" when doing Java dev with Maven way back in the day.

I ripped out the shadowjar stuff and converted both of these conformance programs to use the application plugin. Thanks for the pointer!